.github/workflows/cd.yml

```
@@ -0,0 +1,206 @@
+
+      - name: Install Soroban CLI
+        run: |
+          cargo install --locked soroban-cli --features opt
+
+      - name: Deploy contracts
+        id: deploy
+        env:
+          SOROBAN_ACCOUNT_SECRET: ${{ secrets.SOROBAN_ACCOUNT_SECRET }}
+          NETWORK: ${{ steps.set-network.outputs.network }}
+          RPC_URL: ${{ github.ref == 'refs/heads/main' && 'https://soroban-mainnet.stellar.org' || 'https://soroban-testnet.stellar.org' }}
+        run: |
```

actionlint (1.7.12)

[error] shellcheck reported issue in this script: SC2086:info:12:29: Double quote to prevent globbing and word splitting

(shellcheck)